corehq/apps/data_cleaning/templates/data_cleaning/bulk_edit_main.html

```diff
@@ -24,7 +24,7 @@ <h3 class="card-title">
   <div class="card mt-4">
     <div class="card-body">
       <h5 class="card-title">
-        {% trans "Recent Tasks" %}
+        {% trans "Recent Bulk Edit Sessions" %}
```

Comment:
I guess similar to my other comment, "Recent Sessions"? Don't have strong feelings though, "Recent Bulk Edit Sessions" just feels...bulky...